ic-agent/src/identity/secp256k1.rs

```diff
@@ -23,7 +23,7 @@ use simple_asn1::{
 #[derive(Clone, Debug)]
 pub struct Secp256k1Identity {
     private_key: EcKey<Private>,
-    public_key: EcKey<Public>,
+    _public_key: EcKey<Public>,
```

Comment:
Is this now required to change to _public_key because it's only referenced in tests?

Reply:
Yes, that or #[allow(dead_code)]. In between the versions they removed the exemption for cfg(test) from the normal rules about usedness and cfg.